src/ontology/mondo-edit.obo

```diff
@@ -165475,12 +165470,7 @@ xref: OMIM:158901 {source="MONDO:equivalentTo"}
 xref: Orphanet:269 {source="OMIM:158901"}
 xref: UMLS:C1834671 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:320405"}
 is_a: MONDO:0001347 {source="DC-OMIM:158901", source="DOID:0111193", source="MESH:C563557", source="MONDO:Redundant", source="OMIM:158901", source="Orphanet:269/btnt"} ! facioscapulohumeral muscular dystrophy
-intersection_of: MONDO:0001347 ! facioscapulohumeral muscular dystrophy
-intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/29090 ! SMCHD1
-intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/50800 ! DUX4
 relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/mondo/qc-multiple-gene-associations.sparql
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/29090 {source="MONDO:mim2gene_medgen", source="OMIM:158901"} ! SMCHD1
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/50800 {source="OMIM:158901"} ! DUX4
```

Comment:
This term has the annotation
`excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/mondo/qc-multiple-gene-associations.sparql`

These gene annotations should not be removed.